Dockerfile.centos

```diff
@@ -61,3 +65,8 @@ RUN wget -qO- https://github.com/ccache/ccache/releases/download/v3.7.7/ccache-3
   && make -j \
   && make install \
   && cd ../ && rm -rf ccache-*
+
+# Install lcov to /usr/local/bin
+# require master to make gcc9 happy
+RUN git clone https://github.com/linux-test-project/lcov.git \
```

Comment:
limit branch and depth

Reply:
Can't limit depth which lead a strange error.

Reply:
Can't limit the depth which lead strange error that make vesoft/nebula-dev:centos7 not workable.